### DISCUSSION TITLE
Scoped Webhooks

### DISCUSSION THREAD

**Discussion by axelrindle** [outline/outline#6870] [Ideas]
Creating a Webhook with `All document events` fires for documents in **all collections**.

I'd like to "scope" a Webhook to a certain collection or even a specific document structure within a collection.

Imagine the following structure:

- Project A (*Collection*)
  - Documentation
  - Idea Board
  - ...
- Project B (*Collection*)
  - ...

For example: A scoped Webhook should only trigger for the selected events within `Project A -> Documentation`, e.g. when the page `Documentation` or it's children are updated/deleted/etc.

**Comment by tommoor** [comment]
If someone wishes to contribute this functionality I'd definitely be open to merging. In the meantime it hasn't been high priority as it's generally easy to just discard/ignore events that you don't care about.

**Reply by axelrindle** [reply]
Challenge accepted!
